src/algebra/lie_algebra.lean

```diff
@@ -339,24 +339,31 @@ def lie_ideal_subalgebra (I : lie_ideal R L) : lie_subalgebra R L := {
 
 end lie_module
 
-namespace lie_quotient
+namespace lie_submodule
 
 variables {R : Type u} {L : Type v} [comm_ring R] [add_comm_group L] [lie_algebra R L]
+variables {M : Type v} [add_comm_group M] [module R M] [lie_module R L M]
+variables (N : lie_submodule R L M) (I : lie_ideal R L)
 
 /--
-The quotient of a Lie algebra by a Lie ideal. It is a Lie algebra.
+The quotient of a Lie module by a Lie submodule. It is a Lie module.
 -/
-abbreviation quotient (I : lie_ideal R L) := submodule.quotient I.to_submodule
+abbreviation quotient := N.to_submodule.quotient
+
+namespace quotient
+
+variables {N I}
 
 /--
-Map sending an element of `L` to the corresponding element of `L/I`, when `I` is a lie_ideal of `L`.
+Map sending an element of `M` to the corresponding element of `M/N`, when `N` is a lie_submodule of
+the lie_module `N`.
 -/
-abbreviation mk {I : lie_ideal R L} : L → I.to_submodule.quotient := submodule.quotient.mk
+abbreviation mk : M → N.quotient := submodule.quotient.mk
 
-lemma is_quotient_mk {I : lie_ideal R L} (x : L) :
-  quotient.mk' x = (mk x : quotient I) := by { intros, refl, }
+lemma is_quotient_mk (m : M) :
+  quotient.mk' m = (mk m : N.quotient) := by { intros, refl, }
```

Comment:
What is `intros` doing here? Can't you replace the entire proof by `rfl`?

Reply:
Of course! I don't know how I ended up with that 🤦‍♂